apps/pie-docs/src/_11ty/shortcodes/tokensTable/index.js

```diff
@@ -81,11 +81,26 @@ const buildTokenExampleElement = (token, tokenType, tokenMetadata, path = {}) =>
  * @param {string} globalToken the global token referenced by this alias token
  * @returns a <span> HTML string containing the global token used
  */
-const buildGlobalTokenUsedElement = globalToken => deindentHTML(`
-    <span class="c-tokensTable-tokenDescription">
-        <span class="u-font-bold u-showAboveWide">Global token used:</span>
-        <span class="c-tokensTable-token c-tokensTable-token--light">${globalToken}</span>
-    </span>`);
+const buildGlobalTokenUsedElement = globalToken => {
+    let tokenMarkup = '';
+    let tokenDescription = 'Global token used:';
+
+    if (Array.isArray(globalToken)) {
+        globalToken.forEach(tokenName => {
+            tokenMarkup += `<span class="c-tokensTable-token c-tokensTable-token--light c-tokensTable-token--list">${tokenName}</span>`;
+        });
+
+        tokenDescription = 'Global tokens used:';
+    } else {
+        tokenMarkup += `<span class="c-tokensTable-token c-tokensTable-token--light">${globalToken}</span>`;
+    }
+
+    return deindentHTML(`
```

Comment:
I know its used like this elsewhere but I _think_ you can just indent however you like and not wrap it in this function as the entire shortcode is wrapped in one. Might be worth trying and see?

Reply:
Works. Thanks. Maybe we should look at the other areas too that don't need this so we can clean it up a bit more?